BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemView.swift

```diff
@@ -137,7 +137,7 @@ struct AddEditItemView: View {
                 BitwardenMenuField(
                     title: Localizations.type,
                     accessibilityIdentifier: "ItemTypePicker",
-                    options: CipherType.allCases,
+                    options: CipherType.allCases.filter { $0 != .sshKey },
```

Comment:
🤔 Instead of filtering here, could we possibly put a property on `CipherType` to enumerate all the types that are addable by the mobile app? Just a thought, especially in case we add more types down the line that need to be filtered out in this sort of situation.

Reply:
This makes sense, I'll make the change.